core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*
  * Copyright 2014 Confluent Inc.
```

Comment:
nit: do we change copyright year when touching older files?

Reply:
I actually have no idea @ewen?